public/sitemap.xml

```diff
@@ -0,0 +1,1273 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Should the sitemap be static? How are we updating it when we create or remove content?